src/index.ts

```diff
@@ -321,21 +321,20 @@ export function startAnalysis<
         const harOutputPath = temporaryFile({ extension: 'har' });
 
         const mitmproxyOptions = [
+            '--quiet', // We cannot reliably read the mitmproxy stdout anyway, so we suppress it. (See: https://github.com/tweaselORG/cyanoacrylate/issues/5)
```

Comment:
Are we sure that this will suppress _all_ mitmproxy output? Otherwise, I'd say we should add something like a `cyanoacrylate:` prefix to our messages and only try to parse those.

Reply:
I think this might be the cause of this problem on Windows: https://github.com/tweaselORG/meta/issues/25#issuecomment-1507678704

Reply:
Implemented.